1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -16,12 +16,10 @@ describe("fizzbuzz", () => {
         expect(fizzBuzz(5)).toBe('Buzz');
     })
 
-    it('should return "FizzBuzz" when provided with the value 15', () => {
-        expect(fizzBuzz(15)).toBe('FizzBuzz');
-    })
-    
-    it('should return "FizzBuzz" when provided with the value 45', () => {
-        expect(fizzBuzz(45)).toBe('FizzBuzz')
+    it('should "FizzBuzz" when provided with the values 15 and 45', () => {
+        [15, 45]
+            .map((input:number) => fizzBuzz(input))
+            .forEach((output) => expect(output).toBe('FizzBuzz'));
```

Comment:
you could get rid of the map loop, it's an unnecessary loop here, you can just use forEach, like this:
```ts
it('should return "FizzBuzz" when provided with the values 15 and 45', () => {
  [15, 45].forEach((input) => {
    expect(fizzBuzz(input)).toBe('FizzBuzz');
  });
});
```

Reply:
Resolved. Thank you for the feedback 🙏🏽